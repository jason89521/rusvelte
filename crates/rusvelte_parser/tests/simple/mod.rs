use crate::test_success;

test_success!();
