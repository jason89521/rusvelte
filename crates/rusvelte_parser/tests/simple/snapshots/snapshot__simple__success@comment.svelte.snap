---
source: crates/rusvelte_parser/tests/simple/mod.rs
expression: root
input_file: crates/rusvelte_parser/tests/simple/inputs/comment.svelte
snapshot_kind: text
---
{
  "type": "Root",
  "css": null,
  "start": 0,
  "end": 18,
  "fragment": {
    "type": "Fragment",
    "nodes": [
      {
        "type": "Comment",
        "start": 0,
        "end": 7,
        "data": ""
      },
      {
        "type": "Comment",
        "start": 7,
        "end": 18,
        "data": "data"
      }
    ]
  },
  "module": null,
  "instance": null,
  "options": null
}
